hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -842,7 +842,8 @@ public OzoneBucket getBucketDetails(
         bucket.getQuotaInBytes(),
         bucket.getQuotaInNamespace(),
         bucket.getBucketLayout(),
-        bucket.getOwner()))
+        bucket.getOwner(),
+        bucket.getDefaultReplicationConfig()))
```

Comment:
Thanks for finding it. May be good idea to have simple test?

Otherwise I am +1.